src/main/scala/S3Url.scala

```diff
@@ -23,9 +23,9 @@ class S3Url(url: String) {
   )
 
   val (bucket, key) = url.drop(5).split("/").toList match {
-    case head :: Nil => (head, "")
+    case head :: Nil  => (head, "")
     case head :: tail => (head, tail.mkString("/"))
-    case _ => sys.error(s"unrecognized s3 url: $url")
+    case _            => sys.error(s"unrecognized s3 url: $url")
```

Comment:
Same reason, please revert the changes in this file.

Reply:
I guess I have scalafmt turned on save and commit. Let me rollback to basic formatting.

Reply:
I guess I have scalafmt turned on save and commit. Let me rollback to basic formatting.

Reply:
Done